argoverse/evaluation/detection/eval.py

```diff
@@ -117,7 +118,7 @@ def evaluate(self) -> pd.DataFrame:
         gt_fpaths = list(self.gt_root_fpath.glob("*/per_sweep_annotations_amodal/*.json"))
 
         assert len(dt_fpaths) == len(gt_fpaths)
-        data: DefaultDict[str, List[np.ndarray]] = defaultdict(list)
+        data: DefaultDict[str, List[NDArray[np.float64]]] = defaultdict(list)
```

Comment:
Hi @ahuller3, thanks for your work on this. 

@benjaminrwilson and I were reviewing this today, and we were thinking it might be easier for readability to do something like

if we create something like `argoverse/typing/numpy.py`
```python
NDArrayFloat = Union[NDArray[np.float32], NDArray[np.float64]]
```

What are your thoughts?